### DISCUSSION TITLE
Bug: can't enter into my organizations

### DISCUSSION THREAD

**Discussion by hbatalhaStch** [community/community#61874] [Mobile] [answered]
Whenver I try to get into my organization I get this error:

![Screenshot_20230726-200359_GitHub.jpg](https://github.com/community/community/assets/114578587/928a6faf-64af-49b4-a70a-89fc9b3909e3)

**Comment by dannz510** [comment]
The error message **“invalid user or organization name”** on GitHub usually indicates that the username or organization name you are trying to access is not valid. This could happen for several reasons, such as if the username or organization name has been changed or deleted, if there is a typo in the name, or if the name is not correctly formatted according to GitHub’s naming conventions.

If you are sure that the organization name is correct and properly formatted, it’s possible that the organization has been renamed or deleted. You can try searching for the organization using GitHub’s search function to see if it still exists under a different name. If you are still unable to access the organization, you may want to contact GitHub support for further assistance.

**Reply by hbatalhaStch** [reply]
It's my own organization.

> If you are still unable to access the organization, you may want to contact GitHub support for further assistance.

Might be the option

**Comment by faisalnazir7** [comment]
You might be getting this message while opining the organization in mobile app.
This issue is common, I faced the same problem recently.

**Just update the app and restart the app.**

You will be successfully be able to open your organization in your mobile app.

Hope this will help you.

**Reply by hbatalhaStch** [reply]
I am already on latest version. To be sure I uninstalled and reinstalled it but the issue persists.

**Reply by aconsuegra** [reply]
Hi @hbatalhaStch, sorry about this issue. Could you share here the app version that you have installed?

**Reply by hbatalhaStch** [reply]
![Screenshot_20230728-104205_GitHub.jpg](https://github.com/community/community/assets/114578587/8d627e87-332e-4629-8ef9-0450986573a6)

**Reply by aconsuegra** [reply, marked as answer]
Thanks, the bug was fixed in 1.123, which should be already available in Google Play

**Reply by hbatalhaStch** [reply]
Thanks, I can confirm that it is working now

**Comment by Balaraj96** [comment]
hi am not able to join the organisation link but it is giving an error   Balaraj.Billava@vxceed.com  mail id already in use
